### PR TITLE
Flux incubation proposal

### PR DESCRIPTION
**Pull request by squaremo**
A PR for the purpose of comments etc.

### CODE REVIEW

File: reviews/incubation-flux.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+ * All relevant numbers in terms of users of Flux and Helm Operator
+   (v1) have gone up by a factor around 2x to 4x (production users,
+   users on Slack, mailing list, etc.)
+ * Contributions have gone up in a similar fashion.
+ * Releases have happened in a regular cadence. Our v1 projects now
+   have more documentation and have received a few more features (more
+   below) and many stability improvements.
+ * Our community has matured significantly. We have more project
+   maintainers on board, and we run weekly public meetings. We
+   [codified our governance processes][flux-governance].
```
Comment (scottrigby) [inline reviews/incubation-flux.md]: Should we highlight these as well?
```suggestion
   codified our [governance processes][flux-governance], [contributing guidelines][flux-contributing], and [security processes][flux-security].
```

Comment (squaremo) [reply reviews/incubation-flux.md]: Good suggestion, ta!

File: reviews/incubation-flux.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+ - [Flux in CNCF annual review (2020)][flux-annual]
+
+The Flux project aims to provide a complete Continuous Delivery
+platform on top of Kubernetes, supporting all the common practices and
+tooling in the field, e.g. Helm, kustomize, metrics with Prometheus
+and so on.
+
+Flux was the first project to clearly define [GitOps
+practices][gitops-defn], which can be summarised as:
+
```
Comment (hiddeco) [inline reviews/incubation-flux.md:17]:
```suggestion
```

**Review by hiddeco — APPROVED** [review]

**Review by stefanprodan — APPROVED** [review]
🚀

**Pull request merged**